control_panel_api/elasticsearch.py

```diff
@@ -0,0 +1,26 @@
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Q, Search
+from elasticsearch_dsl.query import Range
+
+
+class ElasticSearch(object):
+    def __init__(self, connection):
```

Comment:
As said in the `views.py` comment, you could make `connection` optional and default to `settings.ELASTICSEARCH_CONN` so client code can be simplified and not pass it unless it really wants to change default.